gtk2_ardour/po/fr.po

```diff
@@ -19760,93 +19754,79 @@ msgstr ""
 msgid "MIDI un-Learn"
 msgstr ""
 
-#: trigger_ui.cc:407
-msgid "Save all MIDI Learn settings"
-msgstr ""
-
-#: trigger_ui.cc:416
-msgid "Choose filename"
-msgstr ""
-
-#: trigger_ui.cc:442
-msgid ""
-"%1 already exists.\n"
-"Please select a different filename"
-msgstr ""
-
-#: trigger_ui.cc:583
+#: trigger_ui.cc:534
```

Comment:
Are you sure that these have been rightly withdrawn?

Reply:
Automatically removed by pot and po file generator for the new ardour 7.1 version.

Reply:
ok